operations/workflow.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+###########################################################
+# Description: Simplifies git workflow on remote machines #
+#                                                         #
+# Usage: ./workflow.sh -opt                               #
+#                                                         #
+# Options: -b or -branch to clone a branch                #
+#          -c or -clean to delete a branch                #
+###########################################################
+
+BRANCH_PATH="$HOME/branch/"
+REPO="https://github.com/bcgov/GDX-Analytics-microservice.git"
+
+while getopts ":b(branch) :c(clean)" opt; do
```

Comment:
Can there be a case where no flags are provided and as a result workflow.sh lists the available opts?

Right now the result is not very explanatory:
```
[microservice@ip-172-31-20-95 branch]$ ./workflow.sh 
[microservice@ip-172-31-20-95 branch]$ 
```

Reply:
This should now be working. If no arg is passed, $OPTIND will equal 1, satisfying the condition and echoing the options